components/resources/library/build.gradle.kts

```diff
@@ -122,12 +132,21 @@ kotlin {
             dependsOn(skikoTest)
             dependsOn(blockingTest)
         }
-        val jsMain by getting {
+        val webMain by creating {
```

Comment:
We need to keep a mirroring of main and test source sets to avoid hidden problems.
need to add the webTest

Reply:
Interesting. I didn't know about hidden problems if sources sets are not mirrored.
Do you know any YT issues or examples?
__

I'll add it.

Reply:
done